applications/FluidDynamicsApplication/custom_processes/two_fluid_navier_stokes_fractional_convection_process.h

```diff
@@ -0,0 +1,646 @@
+            }
+            else
+            {
+                mAuxModelPartName = ThisParameters["ns_frac_convection_model_part_name"].GetString();
+            }
+
+            // Limiter related settings
+            KRATOS_WATCH("")
+        }
+
+        std::string GetNSFractionalConvectionElementRegisteredName(Parameters ThisParameters)
+        {
+            // Convection element formulation settings
+            std::string element_type = ThisParameters["element_type"].GetString();
+            const auto element_list = GetNSFractionalConvectionElementsList();
+            if (std::find(element_list.begin(), element_list.end(), element_type) == element_list.end())
+            {
+                KRATOS_INFO("") << "Specified \'" << element_type << "\' is not in the available elements list. " << "Attempting to use custom specified element." << std::endl;
+                mNSFractionalConvectionElementType = element_type;
+            }
+            else
+            {
+                mNSFractionalConvectionElementType = GetNSFractionalConvectionElementName(element_type);
+            }
+            std::string element_register_name = mNSFractionalConvectionElementType + std::to_string(TDim) + "D" + std::to_string(TDim + 1) + "N";
+            if (!KratosComponents<Element>::Has(element_register_name))
+            {
+                KRATOS_ERROR << "Specified \'" << element_type << "\' is not in the available elements list: " << element_list
+                             << " and it is nor registered as a kratos element either. Please check your settings\n";
+            }
+            return element_register_name;
+        }
+
+        /**
+         * @brief Get the Fractional Velocity Convection Elements List object
+         * This method returns a list with the available formulations for the level set convection
+         * @return const std::vector<std::string> List containing the available formulations
+         */
+        const virtual inline std::vector<std::string> GetNSFractionalConvectionElementsList()
+        {
+            std::vector<std::string> elements_list = {
+                "ns_fractional_velocity_convection"};
+            return elements_list;
+        }
+
+        /**
+         * @brief Get the  Fractional Velocity Convection Element Name object
+         * This method maps the user-defined element name to the Kratos class name
+         * @param InputName User-defined element name
+         * @return const std::string Kratos convection element class name
+         */
+        const virtual std::string GetNSFractionalConvectionElementName(std::string InputName)
+        {
+            const std::map<std::string, std::string> elements_name_map{
+
+                {"ns_fractional_velocity_convection", "TwoFluidNavierStokesFractionalConvection"}
+
+            };
+            return elements_name_map.at(InputName);
+        }
```

Comment:
Considering that we only have one element I'd just hardcode its name and avoid all these complexities.

Reply:
It is true, but in a future I would like to add a new one, with a differente time integration discretization.

Reply:
Fair enough.